module/zfs/vdev_queue.c

```diff
@@ -392,8 +443,10 @@ vdev_queue_class_to_issue(vdev_queue_t *vq)
 	for (p = 0; p < ZIO_PRIORITY_NUM_QUEUEABLE; p++) {
 		if (avl_numnodes(vdev_queue_class_tree(vq, p)) > 0 &&
 		    vq->vq_class[p].vqc_active <
-		    vdev_queue_class_max_active(spa, p))
+		    vdev_queue_class_max_active(spa, vq, p)) {
+			vq->vq_last_prio = p;
```

Comment:
I think the comment on line 133 needs to be updated, because `in the order defined by zio_priority_t` is no longer accurate.  I think that comment would also be a good place to describe the overall impact of min_active, nia_credit, etc.

Reply:
I would not say it is inaccurate.  Priorities are still honored above the min_active values.  I've tuned some comments.